Rakefile

```diff
@@ -1,5 +1,7 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :sle12sp2
```

Comment:
this is wrong. there should not be anything, otherwise public jenkins also send it to SP2